scripts/pi-hole/js/index.js

```diff
@@ -307,7 +307,9 @@ function updateForwardedOverTime() {
         var key, i, j;
         for (key in data.forward_destinations)
         {
-            if (!{}.hasOwnProperty.call(data.forward_destinations, key)) continue;
+            if (!{}.hasOwnProperty.call(data.forward_destinations, key)) {
```

Comment:
above this line, the style is
```
if()
{
    ...
}
```
however your change introduces the form
```
if() {
    ...
}
```

I think the styles should not be mixed. This is equally valid for the remaining changes in this file. My opinion is that `{` should be standalone on its own line if you do the same for `}`

Reply:
Yeah, there were a few other parts of the file with them appended rather than standalone too.

I've gone through now and changed them on if and for statements to standalone for more consistency.

Reply:
Personally, I like the latter form, but it is better to have a consistent form.